### DISCUSSION TITLE
Power BI에서 Map을 사용하는 방법에 대해서 문의드립니다.

### DISCUSSION THREAD

**Discussion by kim-yeon-hong** [fusiondevkr/pl900#82] [1 Q&A]
안녕하세요...^^

Power BI에서 Map을 사용하려니 "맵 및 등치 지역도 시각적 개체는 조직에 사용할 수 없습니다." 라는 메시지가 나옵니다.

해서 Power BI Pro Trail 버전을 신청해서 사용해보려고 해도 역시 마찬가지로 맵이 활성화 되질 않습니다.

바쁘시겠지만 설정의 문제인지 라이선스의 문제인지 확인해주시면 고맙겠습니다.

감사합니다.

**Comment by sesme100** [comment]
테넌트 관리자에게 관리 포털에서 맵과 등치 지역도 시각적 개체를 사용하도록 요청합니다 라는 메세지도 같이 나오지 않나요?

Power BI Desktop의 경우 파일 탭 - 옵션 - 보안- 맵 및 등치 지역 사용 활성화 해주시면 됩니다.

**Reply by sesme100** [reply]
혹시 Power BI Service에서 이런 문제가 나타난 거라면 조직의 관리자가 Admin Center에서 맵 기능을 활성화 시켜주셔야 합니다.
